### ISSUE TITLE
Mouse movement with "Play audio coordinates when mouse moves" enabled fails with exception

### ISSUE THREAD

**Issue by Danstiv** [nvaccess/nvda#18929]
### Steps to reproduce:
1. Go to nvda mouse settings and enable "Play audio coordinates when mouse moves" setting.
2. Try to move the mouse physicly or using nvda mouse movement gestures.

### Actual behavior:

Exception raised, and no beep was played.

### Expected behavior:

Nvda should not raise exception, and mouse audio indication should be played.
### NVDA logs, crash dumps and other attachments:
```
ERROR - core.CorePump.Notify (00:15:10.130) - MainThread (16760):
errors in this core pump cycle
Traceback (most recent call last):
  File "core.pyc", line 994, in Notify
  File "mouseHandler.pyc", line 282, in pumpAll
  File "mouseHandler.pyc", line 224, in executeMouseMoveEvent
  File "mouseHandler.pyc", line 94, in playAudioCoordinates
TypeError: 'RGBQUAD' object is not subscriptable
```
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-52622,2f8a9df9 (2026.1.0.52622)
#### Windows version:
Windows 11 24H2 (AMD64) build 26100.6584
#### Name and version of other software in use when reproducing the issue:
-
#### Other information about your system:
-
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Some old snapshot from 05.08.2025 works correctly.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
-

**Comment by tareh7z** [comment]
Hi!

I tried to reproduce this issue on my setup, but everything works as expected.

Audio coordinates are played when moving the mouse and no exception is raised.

<details>
<summary>NVDA log snippet</summary>

IO - tones.beep (20:02:25.389) - MainThread (11288):
Beep at pitch 383.77777777777777, for 40 ms, left volume 41, right volume 43
IO - speech.speech.speak (20:02:25.394) - MainThread (11288):
Speaking ['list']
IO - speech.speech.speak (20:02:25.395) - MainThread (11288):
Speaking ['Desktop']
IO - tones.beep (20:02:25.420) - MainThread (11288):
Beep at pitch 353.22222222222223, for 40 ms, left volume 42, right volume 42
IO - tones.beep (20:02:25.450) - MainThread (11288):
Beep at pitch 320.83333333333337, for 40 ms, left volume 42, right volume 42
IO - tones.beep (20:02:25.482) - MainThread (11288):
Beep at pitch 278.05555555555554, for 40 ms, left volume 43, right volume 41

</details>

**System info**:
- **NVDA version**: ¨NVDA alpha-52622,2f8a9df9 (installed)  
- **Add-ons**: Works with and without add-ons  
- **OS**: Windows 11 24H2 (build 26100.6584)

Thanks.

**Comment by Danstiv** [comment]
Actually point brightness detection fails, so exception is raised when Play audio coordinates when mouse moves enabled, and Brightness controls audio coordinates volume is also enabled.
I just found this out, also updated STR.
